tests/test_utils/test_transactiontestcase.py

```diff
@@ -26,3 +45,22 @@ def test(self, call_command):
             reset_sequences=False, inhibit_post_migrate=True,
             database='default', verbosity=0,
         )
+
+
+class TestDataConsistencyOnTearDown(TestSerializedContentMockMixin, TransactionTestCase):
+    """
+    TransactionTestCase._fixture_teardown() is flushing the whole database.
+    If an other test after this one needs some data coming from initial data migrations,
+    there data won't be found.
+    One solution is to load again initial data migrations after the flush called on _fixture_teardown().
+    """
+    available_apps = ['test_utils']
+    serialized_rollback = True
+    initial_data_migration = '[{"model": "test_utils.car", "pk": 1, "fields": {"name": "K 2000"}}]'
+
+    def _post_teardown(self):
+        super(TestDataConsistencyOnTearDown, self)._post_teardown()
+        self.assertTrue(Car.objects.exists())
```

Comment:
I understand all that, but I don't see any need for the car instance created by this test case to persistent for other tests? What's the problem with deleting that object as I proposed? Without deleting the object, adding this test will pass:
```python
class TestE(TransactionTestCase):
    available_apps = ['test_utils']

    def test(self):
        self.assertTrue(Car.objects.exists())
```
which I think is unexpected.